src/js/tourFinder.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * The method that get the user Geolocation, if the user open the Tour Finder page.
+ */
+
+$(document).ready(function() {
+    if (window.location.pathname == '/tour-finder.html') {
+        if ('geolocation' in navigator) {
+            //check geolocation available
+            //try to get user current location using getCurrentPosition() method
+            navigator.geolocation.getCurrentPosition(function(position) {
+                getToursIDs(position.coords);
+            });
+        } else {
+            alert("Browser doesn't support geolocation!");
+        }
+    }
+});
+
+/**
+ * The method that get the tours IDs.
+ */
+
+getToursIDs = coords => {
+    fetch(
+        `https://www.outdooractive.com/api/project/api-dev-oa/nearby/tour/jsonp?location=${coords.latitude},${coords.longitude}&radius=5000000`
+    )
+        .then(response => response.json())
+        .then(data => {
+            console.log(data);
+            if (data) {
+                getTours(data.result);
+                console.log('getToursIDs ', data);
+            } else {
+                console.log("can't take data");
+            }
+        });
+};
+
+/**
+ * The method that get the first 3 tours for preview on the Tour Finder Page.
+ */
+
+getTours = result => {
+    fetch(
+        `https://www.outdooractive.com/api/oois/jsonp/${result[0].id},${result[1].id},${result[2].id}?key=yourtest-outdoora-ctiveapi&display=list`
+    )
+        .then(response => response.json())
+        .then(data => {
+            console.log('getTours() ', data);
+            getPoster(data.tour);
+            formatTour(data.tour);
+        });
+};
+
+/**
+ * The method that get the tour poster.
+ * I don't know how to apply to each tour the tour Primary Image. I've try to change the source of the images
```

Comment:
you have to construct a list with all the images, below you return the last value set in the primaryImage, you should return a list, because you have multiple tours